ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -4,10 +4,73 @@
  */
 
 import Controller, { inject as controller } from '@ember/controller';
-import { TAG_TYPE_CONFIG } from 'api/models/worker';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { loading } from 'ember-loading';
+import { notifySuccess, notifyError } from 'core/decorators/notify';
+import { TAG_TYPE_CONFIG, TAG_TYPE_API } from 'api/models/worker';
 
 export default class ScopesScopeWorkersWorkerTagsController extends Controller {
   @controller('scopes/scope/workers/index') workers;
 
+  // =services
+  @service intl;
+
+  // =attributes
+
   tagTypeConfig = TAG_TYPE_CONFIG;
+  tagTypeApi = TAG_TYPE_API;
+
+  @tracked removeModal = false;
```

Comment:
Is this needed? Can't you just key off of `modalTag` to know whether or not the modal should be shown?

Reply:
If we just had one modal that would be possible but since we will have 2 modals on this screen, we need to specify which one to open.